### PR TITLE
Satellitedb refactoring

### PR DESCRIPTION
**Pull request by crawter**

### CODE REVIEW

File: pkg/satellite/satellitedb/dbx/satellitedb.dbx [hunk 1]
```diff
@@ -59,11 +59,11 @@ model project (
...
     field id                   blob
-    field user_id              user.id   cascade
+    field owner_id             user.id   setnull ( nullable, updatable )
 
-    field name                 text ( updatable )
-    field description          text ( updatable )
-    field is_agreed_with_terms bool ( updatable )
+    field name                 text      ( updatable )
+    field description          text      ( updatable )
+    field is_agreed_with_terms bool      ( updatable )
```
Comment (egonelbre) [inline pkg/satellite/satellitedb/dbx/satellitedb.dbx]: Maybe `field terms_accepted` would be somewhat easier to read.

File: pkg/satellite/satellitedb/dbx/satellitedb.dbx.go [hunk 2]
```diff
@@ -672,7 +672,7 @@ func (Company_CreatedAt_Field) _Column() string { return "created_at" }
 
 type Project struct {
 	Id                []byte
-	UserId            []byte
+	OwnerId           []byte
```
Comment (egonelbre) [inline pkg/satellite/satellitedb/dbx/satellitedb.dbx.go]: `OwnerID` and `ID`. https://github.com/golang/go/wiki/CodeReviewComments#initialisms

File: pkg/satellite/satellitedb/dbx/satellitedb.dbx.go [hunk 3]
```diff
@@ -672,7 +672,7 @@ func (Company_CreatedAt_Field) _Column() string { return "created_at" }
...
 	Name              string
 	Description       string
 	IsAgreedWithTerms bool
```
Comment (egonelbre) [inline pkg/satellite/satellitedb/dbx/satellitedb.dbx.go]: `TermsAccepted`

Comment (crawter) [reply pkg/satellite/satellitedb/dbx/satellitedb.dbx.go]: yeah, this name is much more better

**Pull request merged**